test/specs/ellipse.spec.js

```diff
@@ -1,7 +1,21 @@
 describe('Ellipse annotation', function() {
   describe('auto', jasmine.fixtures('ellipse'));
 
-  const options = {
+  // event point callbacks
+  const top = function(xScale, yScale, element, xAdjust, yAdjust) {
+    return {x: element.x + element.width / 2 + xAdjust, y: element.y + yAdjust};
```

Comment:
looks like x/yAdjust are applied in wrong place to start with, if we need to consider those everywhere.

Reply:
no clear what you meant.

Being the top, I move the point in the middle of the width (for x) and at the same y.

Reply:
Actually I thought xAdjust and yAdjust were the options by that name, but are actually just parameters so my thoguht was wrong.